src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -46,8 +46,15 @@ public function getConfigTreeBuilder(): TreeBuilder
      */
     private function addFlashMessageSection(ArrayNodeDefinition $node): void
     {
+        $validFormTypeValues = ['standard', 'horizontal'];
+
         $node
             ->children()
+                ->enumNode('form_type')
+                    ->defaultValue('standard')
+                    ->values($validFormTypeValues)
+                    ->info(sprintf('Must be one of %s', implode(', ', $validFormTypeValues)))
```

Comment:
I was expecting more of a description of what this does, I think `enumNode` should automatically create such a message when using with `config:dump-reference`

Reply:
I used more or less [the explanation from the doc](https://sonata-project.org/bundles/core/3-x/doc/reference/installation.html#configuration).